src/utils/Stores/TransactionsStore.ts

```diff
@@ -0,0 +1,76 @@
+import { action, computed, makeObservable, observable } from "mobx";
+import { TransactionObj } from "../../constants/storeConstants";
+import TransactionObject from "./Modals/TransactionObject";
+
+
+
+class TransactionsStore {
+     transactionsList: Array<TransactionObject> | [] = []
```

Comment:
No need to give type as empty array
This can also be initialized in constructor